front/src/auth_components/templates/reset_password_email/reset_password_email.html

```diff
@@ -0,0 +1,91 @@
+                         width="24" height="24" sodipodi:docname="42_logo.svg"><script xmlns=""/>
+                        <metadata id="metadata17"><rdf:RDF><cc:Work rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
+                                rdf:resource="http://purl.org/dc/dcmitype/StillImage"/></cc:Work></rdf:RDF></metadata>
+                        <defs id="defs15"/>
+                        <sodipodi:namedview pagecolor="#ffffff"
+                                            bordercolor="#666666"
+                                            borderopacity="1"
+                                            objecttolerance="10"
+                                            gridtolerance="10"
+                                            guidetolerance="10"
+                                            inkscape:pageopacity="0"
+                                            inkscape:pageshadow="2"
+                                            inkscape:window-width="1060"
+                                            inkscape:window-height="811"
+                                            id="namedview13" showgrid="false"
+                                            fit-margin-top="0"
+                                            fit-margin-left="0"
+                                            fit-margin-right="0"
+                                            fit-margin-bottom="0"
+                                            inkscape:zoom="0.39642998"
+                                            inkscape:cx="68.450005"
+                                            inkscape:cy="48.350011"
+                                            inkscape:window-x="670"
+                                            inkscape:window-y="233"
+                                            inkscape:window-maximized="0"
+                                            inkscape:current-layer="Calque_1"/>
+                        <g id="g3" transform="translate(-229.2,-372.70002)">
+	                        <polygon
+                                    points="229.2,443.9 279.9,443.9 279.9,469.3 305.2,469.3 305.2,423.4 254.6,423.4 305.2,372.7 279.9,372.7 229.2,423.4 "
+                                    id="polygon5"/>
+                            <polygon
+                                    points="316.1,398.1 341.4,372.7 316.1,372.7 "
+                                    id="polygon7"/>
+                            <polygon
+                                    points="341.4,398.1 316.1,423.4 316.1,448.7 341.4,448.7 341.4,423.4 366.8,398.1 366.8,372.7 341.4,372.7 "
+                                    id="polygon9"/>
+                            <polygon
+                                    points="366.8,423.4 341.4,448.7 366.8,448.7 "
+                                    id="polygon11"/>
+                        </g>
+                    </svg>
+                    Sign in with 42 Intra
+                </button>
+            </div>
```

Comment:
Delete sign in option in forgot password

Reply:
https://github.com/tdameros/42-transcendence/pull/23/commits/645958c5bb58e7adb85ce1a666889ba3ffda1af1